mission/internal_status/src/battery_monitor.py

```diff
@@ -17,28 +18,35 @@ def __init__(self):
 
         # Parameters
         self.path_to_xavier_measurement = rospy.get_param("/battery/xavier/path", default="/sys/bus/i2c/drivers/ina3221x/1-0040/iio:device0/in_voltage0_input")
-        self.path_to_powersense = rospy.get_param("/battery/system/path", default="/dev/ttyUSB1")
+        self.path_to_powersense = rospy.get_param("/battery/system/path", default="/dev/i2c-8")
+        
+        self.i2c_address_powersense_voltage = rospy.get_param("/i2c/psm/address_voltage", default=0x69)
+        self.i2c_address_powersense_current = rospy.get_param("/i2c/psm/address_current", default=0x6a)
+        
         self.critical_level = rospy.get_param("/battery/thresholds/critical", default=13.5)
         self.warning_level = rospy.get_param("/battery/thresholds/warning", default=14.5)
+        
         system_interval = rospy.get_param("/battery/system/interval", 0.05)
         xavier_interval = rospy.get_param("/battery/xavier/interval", 10)
         logging_interval = rospy.get_param("/battery/logging/interval", 10)
 
         # Local variables
         self.xavier_voltage = 0.0
         self.system_voltage = 0.0
+        self.system_current = 0.0
         
         self.xavier_recieved = False
         self.system_recieved = False
 
-        # Power Sense device for system voltage
-        self.powersense_device = serial.Serial(self.path_to_powersense, 115200)
-        self.powersense_device.reset_input_buffer()
+        # Get I2C bus for power sense module
+        self.bus = smbus.SMBus(8)
```

Comment:
Get the bus number with rosparam instead of hard coding it.

Reply:
done